server/bin/pbench-index

```diff
@@ -148,6 +151,9 @@ while read size result ;do
         > $errors_json
     fi
 
+    # Exit if index-pbench fails to clean up properly.
```

Comment:
Why exit? Why not continue with  the next tarball? After all, we are going to wait a minute and do the next tarball when pbench-index is run again in any case - do we expect any difference?

Reply:
So it seems bad in principle to keep processing data when we know we have encountered a problem.  Since this code can't control how it is invoked, we need to solve the larger problem with the cron jobs another way.